test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -5831,6 +5831,17 @@ BOOST_AUTO_TEST_CASE(warn_unused_local_assigned)
 	CHECK_WARNING(text, "Unused");
```

Comment:
Can you please change these two to expect the entire message `"Unused local variable."`?

Reply:
sure